kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/ConfigDrift.java

```diff
@@ -55,15 +60,20 @@ public ConfigDrift() {
     // Changes detected and were found to be un-safe
     UNSAFE_DRIFT,
     // Changes detected, but it wasn't clear if they are safe or not
-    SAFETY_UNKNOWN_DRIFT
+    SAFETY_UNKNOWN_DRIFT,
+    // Changes detected but enforcement for them is unsupported
+    UNSUPPORTED_DRIFT
   }
 
   /**
    * Check if there is a config drift between two topic configuration states.
    *
-   * @param desired the desired {@link ConfiguredTopic}
-   * @param actual an actual {@link ConfiguredTopic}
-   * @param type the checking mode, see {@link Type}
+   * @param desired
```

Comment:
can you roll back these changes? looks like your IDE formatter does not match the existing format.